### PR TITLE
feat(files): bamboo_combination_packs

### PR DESCRIPTION
**Pull request by SquatchHunter**
this pack contains a minor chore from my local branch to amend common npm .gitingore configs to the main devel branch

this PR contains all "as is" combinations for bamboo packs, some textures still require updating and can be either updated in this PR or in a later PR

ticks a box in #71 (does not resolve, still more to do)
resolves: #318 

By checking the following boxes with an X, you ensure that:

- [ ] The pack was tested ingame in at least one device.
- [x] The pack is an existing BT pack, is a missing pack from VT or is an accepted pack/change in a discussion.
- [x] The pack code follows the style guide.
- [x] The commits follow the contribution guidelines.
- [x] The PR follows the contribution guidelines.

- [ ] (Optional) Tested in Windows
- [ ] (Optional) Tested in Android
- [ ] (Optional) Tested in iOS
- [ ] (Optional) Tested in any console
- [ ] (Optional) Tested in BDS

### CODE REVIEW

File: resource_packs/packs.json
Comment (PAPISOP) [inline resource_packs/packs.json:1]: Formatting is not done and id should be the folder path such as `extras/consistent_bamboo/pack_name`

**Review by PAPISOP — CHANGES_REQUESTED** [review]
I will review the packs later

File: resource_packs/packs.json
Comment (SquatchHunter) [reply resource_packs/packs.json:1]: what formatting? I forgot the ids and have fixed them already.

Comment (PAPISOP) [reply resource_packs/packs.json:1]: > what formatting? I forgot the ids and have fixed them already.

there is not a space between `:`, this is how it should look `"key": "value"` but urs look like this `"key":"value"`

**Comment by PAPISOP** [comment]
also the pr doesnt follow contribution guidelines

File: resource_packs/packs.json
Comment (SquatchHunter) [reply resource_packs/packs.json:1]: already fixed.

**Comment by drav0011** [comment]
Not sure of the current state of this PR, any updates?

**Comment by PAPISOP** [comment]
> Not sure of the current state of this PR, any updates?

@SquatchHunter needs to approve my pr or I could just move these combinations to my pr

**Comment by PAPISOP** [comment]
> Not sure of the current state of this PR, any updates?

I have moved the bamboo combinations to my own pr (#385) while co-authoring @SquatchHunter, so @SquatchHunter you may close this pr

**Comment by drav0011** [comment]
Keeping track of this changes in #385

**Pull request closed without merging**